examples/FlinkDeployment.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   image: quay.io/streamshub/flink-sql-runner:v0.0.1
```

Comment:
maybe move it also to :main tag instead of 0.1.1 ?

Reply:
Yeah, in order to use `v2_0` this would need to point to `main`. 

The issue is that there is no guarantee that the `main` image will actually be functional. I wouldn't want someone to come to the repo and try the example to find it doesn't work.

Reply:
ok then we need to release version of functional image